azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -184,10 +184,6 @@ def initialize(self):
 
                 _log_cgroup_info("systemd version: {0}", systemd.get_version())
 
-                # This is temporarily disabled while we analyze telemetry. Likely it will be removed.
```

Comment:
we added those when we started using the 'azure' slice and drop in files... that has been stable for a long time, so no longer needed